mkosi.images/system/mkosi.postinst.chroot

```diff
@@ -65,9 +65,15 @@ if [ -n "$IMAGE_VERSION" ] ; then
 fi
 
 if command -v authselect >/dev/null; then
-    authselect select minimal
+    if [ -d /usr/share/authselect/default/local ]; then
+        PROFILE=local
```

Comment:
can you add a brief comment here along the lines of "authselect 1.50 renamed the 'minimal' profile to 'local', prefer the new name" or so?